### ISSUE TITLE
Automatic separators based on locales ('it) or currency code ('EUR)

### ISSUE THREAD

**Issue by Echecivuole** [autoNumeric/autoNumeric#745]
Hi, I have different currency and locale codes saved in my database, but the current AutoNumeric has no way to change the decimalCharacter and digitGroupSeparator separators just pasting the locale code ('it') or the currency code ('EUR'). How possible?

any solution?

**Comment by AlexandreBonneau** [comment]
Hello @Echecivuole, you can change the decimal character as well as the digit group separator using the aforementioned options, for instance :
```js
{
    decimalCharacter: "⎖",
    digitGroupSeparator: "'"
}
```.

What have you tried?

Could you please create a [reproducible test case](https://codepen.io/pen/) so we can fix your bug?

**Comment by Echecivuole** [comment]
> Hello @Echecivuole, you can change the decimal character as well as the digit group separator using the aforementioned options, for instance :
> 
> ```js
> {
>     decimalCharacter: "⎖",
>     digitGroupSeparator: "'"
> }
> ```
> 
> What have you tried?
> 
> Could you please create a [reproducible test case](https://codepen.io/pen/) so we can fix your bug?

thanks @AlexandreBonneau for the reply! Yes I already appleid this code: 

`$(document).ready(function() {
  var autoNumericInstance = new AutoNumeric('.amount_format', {
    decimalCharacter: ',',
    decimalCharacterAlternative: '.',
    digitGroupSeparator: '.',
    unformatOnSubmit: true,
  });
});`

So it works with static separator. My issue is that this package should has a dynamic way to pass the currency code (eg, USD) and auto-apply separators used by that currency code. The parameter done like "French" or "euro" is not a standard so developers cannot pass automatically.

thanks!

**Comment by AlexandreBonneau** [comment]
Well, this is possible but would need a *database* of all the country codes with their default decimal character, digit group separator and numbers, currency character, etc.
This might be a daunting tasks to do, might add a bit of weight to the library, but that could help this project.

For a quicker way to make it work, we could add an alias for 'it'.
As mentioned in the readme ;
> If you feel a common currency option is missing, please [create a pull request](https://github.com/autoNumeric/autoNumeric/compare) and we'll add it!

**Comment by Echecivuole** [comment]
> Well, this is possible but would need a _database_ of all the country codes with their default decimal character, digit group separator and numbers, currency character, etc. This might be a daunting tasks to do, might add a bit of weight to the library, but that could help this project.
> 
> Right now, you can use predefined options (ie. in [this example](https://codepen.io/AnotherLinuxUser/pen/JjLparb?editors=1010)). For a quicker way to make it work, we could add an alias for `it` there.
> 
> As mentioned in the Readme ;
> 
> > If you feel a common currency option is missing, please [create a pull request](https://github.com/autoNumeric/autoNumeric/compare) and we'll add it!

Yes, in fact such a database is not simple. Most of databases have currency code (EUR, USD, GBP) which declare separators. If you make a update to have something like: 

`var autoNumericInstance = new AutoNumeric('.amount_format', 'USD' )`

so then AutoNumeric automatically gets relevant separators defined by the currency, this js will become powerful! Whitout this, cannot be used with dynamic currencies....:(
